mercury/templates/index.html

```diff
@@ -1,51 +1,69 @@
-          function RedirectSimulator() {
-              window.location="/simulator/";
-          }
-          function RedirectDashboard() {
-              window.location="/dashboard/";
-          }
-          function RedirectCAN() {
-              window.location="/can/";
-          }
-          function RedirectStopwatch() {
-              window.location="/stopwatch/";
-          }
-          function newGrafanaWindow() {
-              window.open("https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-motorsports-telemetry-dashboard?orgId=1&refresh=5s");
-          }
-      </script>
+    <meta charset="utf-8">
+    <title>Mercury - NYU Tandon Telemetry</title>
+    <link rel="icon" href="{% static 'mercury/favicon.png' %}">
+    <script>
+      function RedirectSimulator() {
+        window.location="/simulator/";
+      }
+      function RedirectDashboard() {
+        window.location="/dashboard/";
+      }
+      function RedirectCAN() {
+        window.location="/can/";
+      }
+      function RedirectStopwatch() {
+        window.location="/stopwatch/";
+      }
+      function newGrafanaWindow() {
+        window.open("https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-motorsports-telemetry-dashboard?orgId=1&refresh=5s");
+      }
+    </script>
   </head>
-<body>
-{% include 'banner.html' %}
+  <body>
+  {% include 'banner.html' %}
 
-  <div id="grafana-base">
-    <div id="grafana-panel">
-      <br>
-      <h1>NYU Motorsports Telemetry</h1>
-      <br>
-      <button onclick="newGrafanaWindow()" class="grafana-btn grafana-btn-green" style="width: 50%; height: 80px; font-size: larger;"> Telemetry Dashboard</button>
-      <br>
-      <div style="height: 4px;"></div>
-      <button onclick="RedirectSimulator();" class="grafana-btn grafana-btn-red" style="width: 50%; height: 40px; font-size: larger;">Simulator</button>
-      <div style="height: 4px;"></div>
-      <button onclick="RedirectDashboard();" class="grafana-btn grafana-btn-red" style="width: 50%; height: 40px; font-size: larger;">Raw Telemetry</button>
-      <div style="height: 4px;"></div>
-      <button onclick="RedirectCAN();" class="grafana-btn grafana-btn-red" style="width: 50%; height: 40px; font-size: larger;">CAN Decoder</button>
-      <div style="height: 4px;"></div>
-      <button onclick="RedirectStopwatch();" class="grafana-btn grafana-btn-red" style="width: 50%; height: 40px; font-size: larger;">Stopwatch</button>
-      <br><br>
+    <div id="grafana-base">
+      <div id="grafana-panel">
+
+        <h1>NYU Motorsports Telemetry</h1>
+        <br>
+        This platform allows for monitoring of NYU’s Motorsports. Our simulator generates data such as suspension, temperature, acceleration, wheel speed, and fuel level. The Raw Telemetry section shows the data simulated in tabular format, while the Dashboard shows the data in easy-to-view graphs. Test CAN API by sending messages to the decoder endpoint. Access to the stopwatch to time races.
```

Comment:
I thought we were going to have real descriptions for the site?

Reply:
Working on more detailed descriptions now. Wanted to get the PR in so everything else could be reviewed.